tools/validators/instance_validator/main.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import instance_parser 
+import sys
+# ONTOLOGY VALIDATION
+# import ontology_validation
+# import generate_universe
+
+# TODO add input and return type checks in all functions
+
+if __name__ == '__main__':
```

Comment:
document how to use this.  Note, there are ways to make nice help output.  (first example I found on Google https://stackoverflow.com/questions/9037828/writing-a-help-for-python-script)

Reply:
Done